packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php

```diff
@@ -17,7 +17,7 @@
                 {!! view_render_event('bagisto.shop.categories.toolbar.filter.before') !!}
 
                 <!-- Product Sorting Filters -->
-                <x-shop::dropdown position="bottom-left">
+                <x-shop::dropdown class="z-[1]" position="bottom-left">
```

Comment:
To improve readability, I suggest breaking down your code into multiple lines. If a tag has more than one attribute, it should be formatted over multiple lines

Reply:
Yupp , code updated as per change request .